migrations/versions/0451_create_db_users.py

```diff
@@ -21,6 +21,10 @@ def upgrade():
         op.execute(f"GRANT {super_role} TO {role} WITH ADMIN OPTION;")
 
 
+def downgrade():
+    return
```

Comment:
What if we go all the way and full undo by revoking the grant?
```suggestion
    op.execute(f"REVOKE {super_role} FROM {role};")
```